### DISCUSSION TITLE
Change summary to data_summary in the script resulting from tar_script()

### DISCUSSION THREAD

**Discussion by b-rodrigues** [ropensci/targets#1033] [Ideas]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

Hello

I think that it would be less confusing for new users if `summary` in the pipeline that results from tar_sript() would be renamed to `data_summary` as in here:

https://github.com/b-rodrigues/targets/commit/a0709de8154394eec83507cf9cfa015f230ac563

Maybe a comment like "Type tar_load(data_summary) to load `data_summary`" could also be added.

**Comment by wlandau** [comment]
I agree, should be fixed in 10860809965721cbf41d6b20129eec757b3868bc and 9a6ebf1600eec31d7f83b17fd85c4a443ac6ca54. Thanks.
